src/app/pages/dashboard/widgets/system/common/product-image/product-image.component.spec.ts

```diff
@@ -0,0 +1,3 @@
+describe('ProductImageComponent', () => {
+  // TODO: Write unit tests
```

Comment:
Tests are required to be in the same PR. I think we should at least have a ticket linked in the PR so it's in the queue and doesn't get ignore for a while.